src/UI/admin/components/hall-reservation/AdminHallReservation.js

```diff
@@ -0,0 +1,272 @@
+                              this.setState({
+                                currentReservationActive: hallReservaltions
+                              });
+                            }}
+                          >
+                            <PopUpInfo>
+                              <PopUpInfoText>
+                                {//this.state.dummyDataArray[0].event
+                                //this.state.dummyDataArray[0].hallReservaltions[0]
+                                //.
+                                hallReservaltions.reservationTitle +
+                                  ' ' +
+                                  hallReservaltions.reservationDate +
+                                  ' ' +
+                                  hallReservaltions.reservationStartTime.slice(
+                                    0,
+                                    -3
+                                  ) +
+                                  ' ' +
+                                  hallReservaltions.reservationEndTime.slice(
+                                    0,
+                                    -3
+                                  )}
+                              </PopUpInfoText>
+                            </PopUpInfo>
+                          </ReservationDynamicDiv>
+                        );
+                      }
+                    }
+                  )}
+                </OneDayDivRow>
+              );
+            })}
+          </TableWrapper>
+
+          <Info>
+            <InfoTitle>
+              {
+                //this.state.dummyDataArray[0].event
+                //this.state.dummyDataArray[0].hallReservaltions[0]
+                //.reservationTitle
+                //this.state.currentReservationActive.reservationTitle === undefined ? "WWWWWWWW" : this.state.currentReservationActive.reservationTitle
+                this.state.currentReservationActive.reservationTitle
+                //this.state.currentReservationActive
+              }
+            </InfoTitle>
+            <InfoTime>
+              {//this.state.dummyDataArray[0].startTime.toString().substring(4, 25)
+              this.state.currentReservationActive.reservationDate +
+                ' ' +
+                this.state.currentReservationActive.reservationStartTime.slice(
+                  0,
+                  -3
+                )}{' '}
+              -
+              {//this.state.dummyDataArray[0].endTime.toString().substring(4, 25)
+              this.state.currentReservationActive.reservationEndTime.slice(
+                0,
+                -3
+              )}
+            </InfoTime>
+            <Details
+              img={DetailsIcon}
+              onClick={() => {
+                //console.log(this.state.currentReservationActive.reservationDescription);
+                //this.state.currentReservationActive.infoVisibility = 'visible';
+                this.setState({
```

Comment:
Set-aj state na način na koji sam napisao na slacku (odg. na tvoje pitanje) 
```
{...this.state, 
  currentReservationsActive : {
    ...this.state.currentReservationsActive,
    reservationTitle: bla blblbl,
}}
```

Iako mi nije jasno što ti ovdje točno radiš. Uzmeš value nekog property-ja iz state-a i onda ga assigneaš u isti taj property... npr 
`reservationTitle: this.state.currentReservationActive.reservationTitle,` ???